Documentation/minio-object-store.md

```diff
@@ -56,7 +56,7 @@ kubectl -n rook-minio get objectstores.minio.rook.io
 To check if all the desired replicas are running, you should see the same number of entries from the following command as the replica count that was specified in `object-store.yaml`:
 
 ```console
-kubectl -n rook-minio get pod -l app=minio
+kubectl -n rook-minio get pod -l app=minio,minio_objectstore=my-store
```

Comment:
to simplify, you could remove `app=minio` from the query, right?

Reply:
I would leave it in now, as the second label is only `objectstore` now.